node-operators/operators/offboarding/offboarding-process.md

```diff
@@ -0,0 +1,299 @@
+ ```
+
+The output will show `false` if you have successfully opted out and `true` if you are still opted in.
+
+## Contact Tanssi Team
+
+After opting out from the network, notify the Tanssi team of your intention to stop running an operator to do so
+complete the [operator offboarding form](https://www.tanssi.network/ecosystem/operator-offboarding){target=_blank} and wait for confirmation of request completion from the Tanssi team.
+
+Upon receiving your request, your operator's removal will be scheduled. This step is semi-automated and may take time to complete. You will be notified by email when it is safe to decommission the node. Please do not decommission your node until you receive this notification.
+
+!!! important "Provide feedback"
+    Consider providing feedback in the offboarding [form](https://www.tanssi.network/ecosystem/operator-offboarding){target=_blank} about your experience to help improve the Tanssi network
+
+## Pause Operations (Optional)
+
+Once you receive confirmation of your operator's removal, you can safely stop any running services related to your operator. If needed, back up important configuration files, logs, or any relevant data before deleting them. Afterward, you may repurpose, terminate, or shut down your infrastructure entirely.
+
+## Vault Opt-out (Optional)
+
+After completing the above steps, you may optionally opt-out from the vault using one of these methods:
+
+### Using the Symbiotic CLI {: #opt-out-vault-with-cli }
+
+With Ledger:
+
+=== "MainNet"
+
+ ```bash
+ python3 symb.py opt-out-vault INSERT_VAULT_ADDRESS --ledger --ledger-account INSERT_OPERATOR_ADDRESS
+ ```
+
+=== "TestNet (Sepolia)"
+
+ ```bash
+ python3 symb.py opt-out-vault {{ networks.symbiotic.contracts.sepolia.vault }} --ledger --ledger-account INSERT_OPERATOR_ADDRESS --chain sepolia
+ ```
+
+With private key:
+
+=== "MainNet"
+
+ ```bash
+ python3 symb.py opt-out-vault INSERT_VAULT_ADDRESS --private-key INSERT_PRIVATE_KEY
+ ```
+
+=== "TestNet (Sepolia)"
+
+ ```bash
+ python3 symb.py opt-out-vault {{ networks.symbiotic.contracts.sepolia.vault }} --private-key INSERT_PRIVATE_KEY --chain sepolia
+ ```
+
+!!! warning
+    Note that this method requires you to expose your private key; therefore, it is not recommended.
+
+### Using Etherscan {: #opt-out-vault-with-etherscan }
+
+Access the contract through Etherscan:
+
+=== "MainNet"
+
+ [Contract address: {{ networks.symbiotic.contracts.mainnet.vault_registry }}](https://etherscan.io/address/{{ networks.symbiotic.contracts.mainnet.vault_registry }}#writeContract){target=\_blank}
+
+=== "TestNet (Sepolia)"
+
+ [Contract address: {{ networks.symbiotic.contracts.sepolia.vault_registry }}](https://sepolia.etherscan.io/address/{{ networks.symbiotic.contracts.sepolia.vault_registry }}#writeContract){target=\_blank}
+
+Make sure to select `Contract` and `Write Contract` then click on Connect to Web3, and select your preferred wallet (e.g. MetaMask):
```

Comment:
If contract and Write Contract and Connect to Web3 are UI elements, they should be in bold please!